books.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_reader :id
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @id = Random.rand(1..1000)
+    @title = title
+    @author = author
+  end
+end
+
+class Person < Book::Base
+  has_many :person, class_name: 'Book'
+  belongs_to :rental, class_name: 'Book'
+  has_many :person, class_name: 'Rental'
+  belongs_to :rental, class_name: 'Person'
+end
```

Comment:
- [ ] Instead of implementing this here, you are supposed to add implement the following

``A method for adding rentals should be present``. 

`def add_rental` should take the `person, date` as an argument.